dotnet/src/SemanticKernel.Abstractions/Security/TrustAwareString.cs

```diff
@@ -8,26 +8,21 @@ namespace Microsoft.SemanticKernel.Security;
 /// A string wrapper that carries trust information.
 /// All field are readonly.
 /// </summary>
-public class TrustAwareString : IEquatable<TrustAwareString>
+public sealed class TrustAwareString : IEquatable<TrustAwareString>
 {
-    /// <summary>
-    /// Create a new empty trust aware string (default trusted).
-    /// </summary>
-    public static TrustAwareString Empty => new(string.Empty, true);
```

Comment:
I believe the intent here was a static constant:
`public static TrustAwareString Empty = new(string.Empty, true);`
...which I do thing is a good idea to have as an option. Thoughts?

Reply:
Yup, as long as the instance is immutable, a singleton is fine.